submissions/yanyshpolska/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,220 @@
+                    alt=""
+                  />
+                </div>
+                <span class="popup__item_name">Chat</span>
+              </a>
+              <a href="#" class="popup__item">
+                <div class="popup__item_image_container">
+                  <img
+                    class="popup__item_image"
+                    src="./img/icons/calendar.png"
+                    alt=""
+                  />
+                </div>
+                <span class="popup__item_name">Calendar</span>
+              </a>
+
+              <input
+                class="popup__item_checkbox_input visually-hidden"
+                type="checkbox"
+                id="popup__checkbox"
+              />
+
+              <div class="popup__item_more">
+                <div class="popup__items">
+                  <a href="#" class="popup__item">
+                    <div class="popup__item_image_container">
+                      <img
+                        class="popup__item_image"
+                        src="./img/icons/classroom.png"
+                        alt=""
+                      />
+                    </div>
+                    <span class="popup__item_name">Classroom</span>
+                  </a>
+                  <a href="#" class="popup__item">
+                    <div class="popup__item_image_container">
+                      <img
+                        class="popup__item_image"
+                        src="./img/icons/mars.png"
+                        alt=""
+                      />
+                    </div>
+                    <span class="popup__item_name">Mars</span>
+                  </a>
+                  <a href="#" class="popup__item">
+                    <div class="popup__item_image_container">
+                      <img
+                        class="popup__item_image"
+                        src="./img/icons/apps.png"
+                        alt=""
+                      />
+                    </div>
+                    <span class="popup__item_name">Applications</span>
+                  </a>
+                  <a href="#" class="popup__item">
+                    <div class="popup__item_image_container">
+                      <img
+                        class="popup__item_image"
+                        src="./img/icons/collection.png"
+                        alt=""
+                      />
+                    </div>
+                    <span class="popup__item_name">Collection</span>
+                  </a>
+                </div>
+              </div>
+
+              <label for="popup__checkbox" class="popup__item_checkbox_label">
+                <span class="popup__item_checkbox_more">More</span>
+              </label>
+            </div>
+          </div>
+        </div>
+
+        <a class="menu__item_picture menu__item" href="#">
+          <img
+            class="menu__item_picture_img"
+            src="./img/bell.png"
+            alt="Alerts"
```

Comment:
https://www.a11yproject.com/posts/alt-text/

Reply:
For this particular case, I believe that usage of the "alt" is reasonable: in case the picture is not loaded the user is still able to see picture caption which is aligned with advice from this article: 
"Use the alt attribute for images that are used as more than decoration."

Reply:
Yes, it is reasonable, but the value is not correct